src/components/teams/Teams.js

```diff
@@ -245,30 +243,20 @@ export default class Teams extends Component {
   }
 
   _getTeamsNavigationTab(team) {
-    var dangerZone = team.ranking.length - 2;
-    var ranking = this._renderOwnTeamPosition(team);
-    var style = this._getLabelClassName(ranking,dangerZone);
+    var style = this._getLabelClassName(team);
     return ( <NavItem eventKey={team.id}>
           <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
-          <span className={style}>{ranking}</span></div>
+          <span className={style}>{this._renderOwnTeamPosition(team)}</span></div>
       </NavItem>
     );
   }
 
-  _getLabelClassName(ranking,dangerZone) {
-    if (this._isTopper(ranking)) {return 'label label-as-badge label-success';}
-    if (this._isInDegradationZone(ranking,dangerZone)) {return 'label label-as-badge label-danger';}
+  _getLabelClassName(team) {
+    if (team.isTopper(this._renderOwnTeamPosition(team),3)) {return 'label label-as-badge label-success';}
```

Comment:
`team.isTopper()` --> als je de 3 als parameter meegeeft en je wil dit veranderen, dan moet je toch nog overal de 3 gaan zoeken en wijzigen. Als je de 3 in de functie zelf stopt, dan moet je de 3 maar op 1 plaats wijzigen. En dan betekent 'topper' hetzelfde overal op de site.

de andere parameter: `this._renderOwnTeamPosition(team)`

```
_renderOwnTeamPosition(team) {
    const ranking = team.getDivisionRanking();
    return ranking ? ranking.position : '?';
  }
```

Deze hoeft ook niet als parameter meegegeven worden, de call `team.getDivisionRanking()` wordt dan gewoon

```
const ranking = this.getDivisionRanking();
if (!ranking || ranking.position) {
  return false;
}
if (gamesPlayer < 5) {
  return false; // deze logica is ook al aanwezig in TeamModel (als er nog maar 1 match gespeeld is en dat was tegen de eerste en je verliest, daarom zit je nog niet in de degradatiezone...)
}
return ranking.position <= 3;
```